Gems/AtomLyIntegration/CommonFeatures/Code/Source/Mesh/MeshComponentController.cpp

```diff
@@ -286,6 +292,11 @@ namespace AZ
             if (m_meshFeatureProcessor)
             {
                 m_meshFeatureProcessor->SetTransform(m_meshHandle, world, m_cachedNonUniformScale);
+                // ensure the render geometry is kept in sync with any changes to the entity the mesh is on
+                AzFramework::RenderGeometry::IntersectionNotificationBus::Event(
+                    FindOwningContextId(m_entityComponentIdPair.GetEntityId()),
```

Comment:
The context shouldn't change after activation, so it may be worth caching that so you don't need to look it up again.

Reply:
Good shout! @AMZN-AlexOteiza mentioned something very similar too 😝 I'll update this, going to try and bind the actual bus function which might be faster still. Cheers! 👍

Reply:
Now updated